nncf/quantization/layers.py

```diff
@@ -307,22 +311,40 @@ def run_export_quantization(self, x: torch.Tensor):
             input_high = input_range
 
             if self._export_mode == QuantizerExportMode.ONNX_QUANTIZE_DEQUANTIZE_PAIRS:
-                y_scale, y_zero_point = get_scale_zp_from_input_low_input_high(self.level_low,
-                                                                               self.level_high,
-                                                                               input_low,
-                                                                               input_high)
+                if self.is_saturation_fix:
+                    y_scale, y_zero_point = get_scale_zp_from_input_low_input_high(2 * self.level_low,
+                                                                                   2 * self.level_high + 1,
+                                                                                   2 * input_low,
+                                                                                   127. / 63. * input_high)
```

Comment:
The last argument recalculation will only work for 256 levels, while the other 3 will seemingly work for all values of levels. Avoid the 256-level limitation in this calculation. 

In general, having such magical constants in code is a bad idea, anywhere.

Reply:
Agree. Please take a look at the new version